gtk/Session.cc

```diff
@@ -308,13 +308,30 @@ void Session::Impl::dec_busy()
 namespace
 {
 
-bool is_valid_eta(int t)
+template<typename T>
+// NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
+constexpr int compare_generic(T const& a, T const& b)
+{
+    if (a < b)
+    {
+        return 1;
+    }
+
+    if (a > b)
+    {
+        return 1;
+    }
```

Comment:
one of the two should be:
```
return -1;
```

Reply:
Fixed in 36a05ee619a3a3a09979d3dfce6b9a46fc616e55

Reply:
> return a < b ? -1 : (a > b ? 1 : 0);

That restores the sonarcloud code smell about nesting conditional operators.